### PR TITLE
Use CPU-only version of PyTorch

### PR DESCRIPTION
**Pull request by ralf-berger**
This PR originally meant to switch from a GPU to a CPU version of PyTorch.

In it current state it does not do that, but:
* downgrades PyTorch
* upgrades lots of other dependencies
* contains information about an unrelated issue

### CODE REVIEW

File: backend/5-studycompass/Pipfile.lock
Comment (ralf-berger) [inline backend/5-studycompass/Pipfile.lock:1]: Lots of unrelated updates

**Comment by shoebjoarder** [comment]
The frontend-1 is sending a request to itself
```
POST http://localhost:8080/api/auth/login/ 405 (Not Allowed)
```
It should be able to send a request to the `backend/2-api-gateway` service which is running at port 8000

**Comment by ralf-berger** [comment]
> The frontend-1 is sending a request to itself
> 
> ```
> 
> POST http://localhost:8080/api/auth/login/ 405 (Not Allowed)
> 
> ```
> 
> It should be able to send a request to the `backend/2-api-gateway` service which is running at port 8000

Did you mean to create an issue or is this somehow related to the PR?

**Comment by shoebjoarder** [comment]
> Did you mean to create an issue or is this somehow related to the PR?

The build was successful in this PR. However, I faced this issue while testing the application, to check whether all services are communicating with each other or not. We can create an issue regarding this if you like.

**Review by shoebjoarder — APPROVED** [review]

File: backend/5-studycompass/Pipfile.lock
Comment (shoebjoarder) [reply backend/5-studycompass/Pipfile.lock:1]: What seems to be the issue?

**Pull request merged**